baton_description:
  name: blue_baton
  radius: 0.01
  length: 0.12
  #HSV scales
  #  hue: 0.0 -> 360.0
  #  sat: 0.0 -> 1.0
  #  val: 0.0 -> 1.0
  hsv_min:
    hue: 230.0
    sat: 0.3
    val: 0.3
  hsv_max:
    hue: 250.0
    sat: 1.0
    val: 1.0

hough_detector:
  accum_scale: 2 4 #Invearse scaling for accumulator size (larger allows more approximate circles)
  min_dist_scale: 4 #Invearse scaling for minimum distance between circles (larger lets detections be closer)
  param1: 100 #Hough specific parameters
  param2: 40 #Hough specific parameters
  min_radius: 50 #Minimum circle detection radius (0 to unset)
  max_radius: 400 #Maximum circle detection radius (0 to unset)

filters:
  openning:
    size: 3
  closing:
    size: 3
  blur:
    size: 9
    distribution: 2
  pose_low_pass:
    size_alpha: 0.1
    position_alpha: 0.2

camera_is_rectified: false
